randomizer/Lists/KasplatLocations.py

```diff
@@ -250,6 +250,7 @@ def setKasplat(self, state=True):
             zmin=1525,
             zmax=1590,
             region=Regions.AngryAztecOasis,
+            additional_logic=lambda l: l.vines or l.jetpack,
```

Comment:
Needs to be `l.vines or (l.jetpack and l.isdiddy)`
This is because jetpack is only valid if and only if the kasplat is for diddy